api/client.ml

```diff
@@ -213,4 +215,10 @@ module Commit = struct
          | Pending -> Ok `Pending
          | Undefined i ->
              Error (`Msg (Fmt.str "client.states: undefined state %d" i)))
+
+  let message t =
+    let open Raw.Client.Commit.Message in
+    let request = Capability.Request.create_no_args () in
+    Capability.call_for_value t method_id request
+    >>= (Results.message_get >> fun x -> Ok x)
```

Comment:
```suggestion
    >>= (Results.message_get >> Result.ok)
```
Only a suggestion; feel free to ignore it.

Reply:
Good suggestion, didn't know that existed:)